docs/package.json

```diff
@@ -77,28 +77,28 @@
     "json2mq": "^0.2.0",
     "jss": "^10.0.0",
     "jss-plugin-template": "^10.0.0",
-    "jss-rtl": "^0.2.3",
+    "jss-rtl": "^0.3.0",
     "lodash": "^4.17.15",
     "lz-string": "^1.4.4",
     "markdown-to-jsx": "^6.10.2",
     "marked": "^0.7.0",
     "material-table": "^1.50.0",
     "material-ui-popup-state": "^1.4.1",
     "next": "8.1.0",
-    "notistack": "^0.8.9",
+    "notistack": "^0.9.2",
     "nprogress": "^0.2.0",
     "postcss": "^7.0.18",
     "prismjs": "^1.17.1",
     "prop-types": "^15.7.2",
-    "raw-loader": "^1.0.0",
+    "raw-loader": "^3.1.0",
     "react": "^16.10.1",
     "react-autosuggest": "^9.4.3",
     "react-docgen": "^5.0.0-beta.1",
     "react-dom": "^16.10.1",
-    "react-draggable": "^3.0.5",
+    "react-draggable": "^4.0.0",
     "react-final-form": "^6.3.0",
     "react-frame-component": "^4.1.1",
-    "react-inspector": "^3.0.2",
+    "react-inspector": "^4.0.0",
```

Comment:
What insight did you gain over https://github.com/mui-org/material-ui/pull/17605?

Reply:
It seems to break SSR too.

I'm not sure about using our tree view, it seems that a non-obvious layer of UI is required to reproduce Chrome output.